### ISSUE TITLE
Allow "panic button" apps to lock KeePass DX

### ISSUE THREAD

**Issue by J-Jamet** [Kunzisoft/KeePassDX#129]
linked to https://github.com/bpellin/keepassdroid/pull/72

@eighthave I see the pull request in KeePassDroid since 2015 and it is not yet merged, can you tell me if the "panic button" is still maintained, because it could be interesting to integrate it with KeePass DX, but there were several dead links. Thank you

**Comment by eighthave** [comment]
yes, it is maintained!  The core parts of the panic library are really
simple, so they haven't needed updating since they were created.  let me
know about dead links, there shouldn't be any, but I might have messed
up the migration away from Redmine.

**Comment by J-Jamet** [comment]
OK, very cool! :)
For dead links : 
 - https://panicbutton.io/ (maybe normal if unsupported)
 - https://dev.guardianproject.info/projects/panic/wiki/PanicButton_wizard_screenshots (in https://dev.guardianproject.info/projects/panic/wiki/ -> Panic Button Wizard Screenshots)
 - All links in https://dev.guardianproject.info/projects/panic/wiki/Wiki/ (from "wiki" (root) in https://dev.guardianproject.info/projects/panic/wiki/)
 - All menu links after a click on "issues" (wrong path, add issues folder)
